apps/mobile/src/libs/styles/global-styles/flex.ts

```diff
@@ -0,0 +1,34 @@
+const FLEX = {
+	alignItemsCenter: {
+		alignItems: "center" as const,
+	},
+	alignItemsEnd: {
+		alignItems: "flex-end" as const,
+	},
+	alignItemsStart: {
+		alignItems: "flex-start" as const,
+	},
+	flex1: {
+		flex: 1,
+	},
+	flexDirectionColumn: {
+		flexDirection: "column" as const,
+	},
+	flexDirectionRow: {
+		flexDirection: "row" as const,
+	},
+	justifyContentCenter: {
+		justifyContent: "center" as const,
+	},
+	justifyContentEnd: {
+		justifyContent: "flex-end" as const,
+	},
+	justifyContentSpaceBetween: {
+		justifyContent: "space-between" as const,
+	},
+	justifyContentStart: {
+		justifyContent: "flex-start" as const,
+	},
+};
```

Comment:
And you can remove all `as const`

```suggestion
} satisfies GlobalFlexStyles;
```